docs/release-notes/release-notes-0.9.5.md

```diff
@@ -0,0 +1,190 @@
+(Note that the master branch of python-bitcointx now has full taproot support, i.e. constructing and spending taproot outputs, also; but we don't yet have a use case for that, anyway).
+Users can now send to taproot addresses via any of the user interfaces (CLI, Qt GUI, web UI).
+
+`b700f37` Add send-to-P2TR
+
+`a467ec4` upgrade to python-bitcointx 1.1.3
+
+### Extensive updates to the JSON RPC-API (jmwalletd.py); first fully working version
+
+After a lot of testing and bugfixing, the following commits were added to patch up the JSON RPC-API feature that was added in [0.9.3](https://github.com/JoinMarket-Org/joinmarket-clientserver/releases/tag/v0.9.3). The new [webUI project](https://github.com/joinmarket-webui/joinmarket-webui) is using this and is now functional (alpha release coming shortly).
+
+For those not familiar, the RPC API can be served by running the script `jmwalletd.py` and note that unlike other Joinmarket scripts, a wallet need not be specified, since it will allow the unlocking of any wallet (or creation of a new one) from a calling RPC client. See the [user docs](https://joinmarket-org.github.io/joinmarket-clientserver/JSON-RPC-API-using-jmwalletd.html) and [API docs](https://joinmarket-org.github.io/joinmarket-clientserver/api/) for details.
+
+There are several commits, broken into two groups here:
+
+#### New endpoints/functionality
+
+`29b459a` Add freeze method to RPC-API
+`89896e0` Add yieldgen report endpoint to API
+`6e07e4f` Add /taker/stop endpoint to RPC
+`28fdaa1` Allow re-unlock of wallets via /unlock
+`3939714` Add status, label and extradata to RPC display
+`e598b35` Add RPC API endpoint for showseed
+
+#### Fixes to bugs in logic, improvements
+
+`a0b7b38` Return TransactionFailed when insufficient funds
+`30e96f8` Do not call stopService at exit in RPC daemon
+`7b822a4` Do not update coinjoin state if maker cannot start
+`d493343` Ensure coinjoin state is reset if wallet switches.
+`4389338` Make unconf unspent display default in jmwalletd
+`ed7b4e1` Fix bugs in unlock-lock logic
+`beaa8b0` fixes to API and return format
+`d4d3157` Unlock does not block waiting for sync
+`bd33b6d` RPC maker/start returns 409 error if no coins
+`e6ea9d0` Update HTTP status codes returned by RPC-API
+`fe46b7e` Allow RPC coinjoin taker_finished to accept unconf
+`66d7e46` Fix RPC timelockaddress call.
+
+
+### Changes to dependencies
+
+Since the introduction of the use of python-bitcointx in 0.7.0, we have effectively been using two different bindings to `libsecp256k1`, i.e. there was still a dependency on coincurve even though we only used it for one or two minor things. This was obviously very suboptimal, and it has now been removed. We also remove a dependency on `libgmp` because `libsecp256k1` itself removed this dependency. Note that `68a426a` was effectively overriden by the taproot update in `a467ec4` but this has no functional effect.
```

Comment:
```suggestion
Since the introduction of the use of python-bitcointx in 0.7.0, we have effectively been using two different bindings to `libsecp256k1`, i.e. there was still a dependency on coincurve even though we only used it for one or two minor things. This was obviously very suboptimal, and it has now been removed. We also removed a dependency on `libgmp` because it doesn't exist as dependency in `libsecp256k1`anymore. Note that `68a426a` was effectively overriden by the taproot update in `a467ec4` but this has no functional effect.
```

Reply:
Tbh I don't quite understand either of these? (not that they really matter to me, either way). Surely present simple tense 'remove' is fine here, as it's something updated *in* this release, i.e. now, to most users (sure, for power users that run off master, it happened in the past). And what's the difference between 'it doesn't exist in .. anymore' and 'it was removed from ...'?

Reply:
I'm pushing this out now so I'll leave these changes not included, hope that's OK.